frame/staking/src/pallet/mod.rs

```diff
@@ -559,6 +651,184 @@ pub mod pallet {
 	#[pallet::getter(fn current_planned_session)]
 	pub type CurrentPlannedSession<T> = StorageValue<_, SessionIndex, ValueQuery>;
 
+	/// Wrapper struct for Era related information. It is not a pure encapsulation as these storage
```

Comment:
Would be good if you enumerate which storage items exactly you think should only be accessed thourgh this type. Once you have that list, you can look into the code and actually see if you are abiding by it or not :)

Reply:
Do you think I should move all of it in one go? I enumerated it previously but then I thought maybe we should keep moving all era related storage items here as we encounter them.